atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -132,63 +134,57 @@ def generate_auto_labels(self, auto_labels):
             generated['vcs-url'] = vcs.vcs_url
             generated['vcs-ref'] = vcs.vcs_ref
 
-        for old, new in self.aliases.items():
-            self.log.info("old=%r new=%r", old, new)
-            if new in generated and old not in generated:
-                self.log.info("adding %r for compatibility", old)
-                generated[old] = generated[new]
+        for lbl in self.auto_labels:
+            if lbl not in generated:
+                self.log.warning("requested automatic label %r is not available", lbl)
```

Comment:
well this was there before, and it might happen that you specify in auto_labels some labels, but you don't generate it, so it won't be found in generated